File: 01gpio/01gpio/led_wiring.py
import wiringpi as wpi
import time

PIN_LED = 18
wpi.wiringPiSetupGpio()
wpi.pinMode(PIN_LED, wpi.OUTPUT)

for i in range(5):
    wpi.digitalWrite(PIN_LED, True)
    print( 'pin %s ON'%PIN_LED)
    time.sleep(1)
    wpi.digitalWrite(PIN_LED, False)
    print( 'pin %s Off'%PIN_LED)
    time.sleep(1)
wpi.digitalWrite(PIN_LED, False)
wpi.pinMode(PIN_LED, wpi.INPUT)



    

File: 01gpio/01gpio/led_wiring.js
//npm install node-wiring-pi
var wpi = require('node-wiring-pi'); 
//var wpi = require('wiring-pi'); //install fail

wpi.wiringPiSetupGpio();
wpi.pinMode(18, wpi.OUTPUT);

for(var i=0; i<10; i++){
	wpi.digitalWrite(18, wpi.HIGH);
	console.log("LED On");
	wpi.delay(500);
	wpi.digitalWrite(18, wpi.LOW);
	console.log("LED Off");
	wpi.delay(500);
}